app/src/pages/inside/common/stackTrace/stackTrace.jsx

```diff
@@ -104,6 +105,13 @@ export class StackTrace extends Component {
     this.fetchItems();
   }
 
+  componentDidUpdate(prevProps) {
+    const { items, logItem } = this.props;
+    if (this.isItemsExist() === 0 && prevProps.items !== items && logItem.status === FAILED) {
+      this.fetchItems();
+    }
```

Comment:
I believe we can attach an error log to any test item or rewrite the status of a test item. So maybe `status===failed` is not the best or universal solution. Please discuss with the lead whether we will support such a case or not.

Reply:
@oranje322 Not only failed items have error logs, it's true.